query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/schema/CreateTypeStart.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.querybuilder.schema;
+
+public interface CreateTypeStart extends OngoingCreateType   {
+
+
+    /**
+     * Adds 'IF NOT EXISTS" to the create type specification. This indicates that the type should
+     * not be created it already exists.
+     */
+    CreateTypeStart ifNotExists();
```

Comment:
This might also apply to other schema types as well, will look into it 👍

Reply:
I just realized after making this change that we still want it to return a `Start` type because we don't want the query to be buildable until we've specified at least a field, so I'll revert thngs back.